kuma/static/js/wiki-skinny.js

```diff
@@ -0,0 +1,724 @@
+            });
+        } else {
+            // generic error recorded - no details if user not logged in
+            mdn.analytics.trackError('Kumascript Error', 'generic error');
+        }
+    }
+
+    /*
+        Track untranslated pages
+    */
+    // is there an translation banner?
+    var $docPending = $('#doc-pending-fallback');
+    if ($docPending.length){
+        mdn.analytics.trackError('Translation Pending', 'displayed');
+    }
+
+    /*
+        Stack overflow search form, used for dev program
+        ex: http://stackoverflow.com/search?q=[firefox]+or+[firefox-os]+or+[html5-apps]+foobar
+    */
+    $('.stack-form').html('<form action="http://stackoverflow.com/search"><i class="stack-icon" aria-hidden="true"></i><label for="stack-search" class="offscreen">' + gettext('Search Stack Overflow') + '</label><input id="stack-search" placeholder="' + gettext('Search Stack Overflow') + '" /><button type="submit" class="offscreen">Submit Search</button></form>').find('form').on('submit', function(e) {
+        e.preventDefault();
+
+        var value = $(this).find('#stack-search').val();
+
+        if (value !== '') {
+            win.location = 'http://stackoverflow.com/search?q=[firefox]+or+[firefox-os]+or+[html5-apps]+' + value;
+        }
+    });
+
+
+    /*
+       Bug 981409 - Add some CSS fallback for browsers without MathML support.
+
+       This is based on
+       https://developer.mozilla.org/en-US/docs/Web/MathML/Authoring#Fallback_for_Browsers_without)MathML_support
+       and https://github.com/fred-wang/mathml.css.
+    */
+    if ($('math').length) {
+        (function() {
+            // Test for MathML support
+            var $div = $('<div class="offscreen"><math xmlns="http://www.w3.org/1998/Math/MathML"><mspace height="23px" width="77px"/></math></div>').appendTo(document.body);
+            var box = $div.get(0).firstChild.firstChild.getBoundingClientRect();
+            $div.remove();
+
+            var supportsMathML = Math.abs(box.height - 23) <= 1 && Math.abs(box.width - 77) <= 1;
+            if (!supportsMathML) {
+                // Add CSS fallback
+                $('<link href="' + mdn.staticPath + 'styles/libs/mathml.css" rel="stylesheet" type="text/css" />').appendTo(doc.head);
+
+                // Add notification
+                $('#wikiArticle').prepend('<div class="notice"><p>' + gettext('Your browser does not support MathML. A CSS fallback has been used instead.') + '</p></div>');
+            }
+        })();
+    }
+
+    /*
+      Make Compare selected revisions button sticky when scrolling history page
+    */
+    (function() {
+        var $button = $('.revision-list-controls .link-btn');
+        if ($button.length) {
+            var revisionButtonOffset = $button.offset().top;
+            $(win).on('scroll', function() {
+                var $compareButton = $button;
```

Comment:
Is there a specific reason for this line? Any reason we cannot simply use `$button.toggleClass...` on line 429?